### ISSUE TITLE
Turn off useSuffixPatternMatching by default?

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#826]
``spring-projects/spring-framework#23915``
``spring-projects/spring-framework#24179``

The current URL mapping doesn't seem to be in good shape for secirity and performance.

___

Explore points to keep in mind.
ex)CsrfSecurityRequestMatcher, ServletConfiguration

The question of how to migrate an implementation that relies on URL pattern matching.
Migrating to AntPathMatcher is recommended, but new issues may occur.

Although REST is not explicitly stated in the specification as a url with an extension, it is used in the example.
In short, a simple migration of AntPathMatcher can be difficult.
If there is a good way to solve this problem without the controller bloating.

**Comment by tesshucom** [comment]
Solved in #1041.

**Comment by tesshucom** [comment]
\
